website/src/pages/Projects.tsx

```diff
@@ -0,0 +1,343 @@
+import { useState, useEffect } from "react";
+import { projects, cases, challenges, Project } from "@/constants/projects";
+import { ProjectDialog } from "@/components/ProjectDialog";
+import { Card, CardContent, CardDescription, CardHeader, CardTitle } from "@/components/ui/card";
+import { useParams, useNavigate } from "react-router-dom";
+import { Trophy, Award, Lock } from "lucide-react";
+import Navbar from "@/components/Navbar";
+import Footer from "@/components/Footer";
+import { Tooltip, TooltipContent, TooltipProvider, TooltipTrigger } from "@/components/ui/tooltip";
+import { MAIN_SPONSOR, CASE_SPONSORS, CHALLENGE_SPONSORS, TECH_SPONSORS, FOOD_SPONSORS, VENUE_SPONSORS } from "@/constants/partners";
+
+// Define the order of cases
+const caseOrder = ['trade-republic', 'avi', 'beam'] as const;
+
+// Define the order of challenges
+const challengeOrder = ['mistral', 'tanso', 'beyond-presence', 'visionaries'] as const;
+
+const ChallengePlaceholder = () => (
+    <div className="flex flex-col items-center justify-center p-8 text-center">
+        <div className="relative">
+            <div className="absolute inset-0 blur-xl opacity-30" style={{
+                background: 'linear-gradient(45deg, #eaf3fd, #a8d0ff)',
+            }} />
+            <div className="relative flex flex-col items-center gap-2">
+                <Lock className="w-6 h-6 text-gray-400" />
+                <span className="text-sm text-gray-400">To be announced</span>
+            </div>
+        </div>
+    </div>
+);
+
+export default function Projects() {
+    const { projectId } = useParams();
+    const navigate = useNavigate();
+    const [selectedProject, setSelectedProject] = useState<Project | null>(null);
+
+    // Group and sort projects by case
+    const projectsByCase = projects.reduce((acc, project) => {
+        if (!acc[project.case]) {
+            acc[project.case] = [];
+        }
+        acc[project.case].push(project);
+        return acc;
+    }, {} as Record<string, Project[]>);
+
+    // Sort projects within each case
+    Object.keys(projectsByCase).forEach(caseKey => {
+        projectsByCase[caseKey].sort((a, b) => {
+            // First sort by placement (1st place, then 2nd place)
+            if (a.placement && b.placement) {
+                return a.placement - b.placement;
+            }
+            if (a.placement) return -1;
+            if (b.placement) return 1;
+
+            // Then sort challenge winners
+            const aHasChallenges = a.challenges && a.challenges.length > 0;
+            const bHasChallenges = b.challenges && b.challenges.length > 0;
+            if (aHasChallenges && !bHasChallenges) return -1;
+            if (!aHasChallenges && bHasChallenges) return 1;
+
+            // Finally sort alphabetically by name
+            return a.name.localeCompare(b.name);
+        });
+    });
```

Comment:
This could be optimized to not do everything on each rerender & go over all projects twice + nested sort. It shouldn't be to bad for the amount of projects we will have, but not sure how they will be loaded & if there might be a few more rerenders in the future.